db/model/room.js

```diff
@@ -75,8 +80,28 @@ module.exports = function room(seq, dataTypes) {
         return RoomType.findById(instance.type)
         .then((roomType) => {
           instance.settings = roomType.settings;
+          const changedKeys = Object.keys(instance._changed);
+          const ignoreAttributes = ['isDeleted'];
+          return common.publishChange(instance, roomEventNames.add, changedKeys,
+              ignoreAttributes);
         });
       },
+
+      afterUpdate(instance /* , opts */) {
+        if (instance.changed('settings')) {
+          if (instance.active) {
+            common.publishChange(instance, roomEventNames.upd);
```

Comment:
Once you have made the changes to the "publishObject" method in /realTime/redisPublisher.js, call it here.